src/stream/TimeshiftBuffer.cpp

```diff
@@ -165,6 +165,8 @@ void TimeshiftBuffer::RemoveOldestInMemoryAndOnDiskSegments()
   m_minInMemorySeekTimeIndex = m_segmentTimeIndexMap.cbegin()->first;
 
   Log(LOGLEVEL_DEBUG, "%s - Removed oldest in memory segment with ID: %d", __FUNCTION__, oldFirstSegment->GetSegmentId());
+  Log(LOGLEVEL_DEBUG, "%s - Removed oldest on disk segment CHECK enabled: %d, paused: %d - segmentTotalCount: %d, maxOnDiskSegments: %d, currentDemuxTimeIndex: %d, minOnDiskSeekTimeIndex: %d", __FUNCTION__,
+                      m_enableOnDiskSegmentLimit, m_paused, m_segmentTotalCount, m_maxOnDiskSegments, m_currentDemuxTimeIndex, m_minOnDiskSeekTimeIndex);
```

Comment:
Really long line :smile: 
Could it make sense to make in several lines? E.g.:
```cpp
  Log(LOGLEVEL_DEBUG, "%s - Removed oldest on disk segment CHECK:", __func__);
  Log(LOGLEVEL_DEBUG, " - enabled:                %s", m_enableOnDiskSegmentLimit ? "yes", "no");
  Log(LOGLEVEL_DEBUG, " - paused:                 %s", m_paused ? "yes", "no");
  Log(LOGLEVEL_DEBUG, " - segmentTotalCount:      %d", m_segmentTotalCount);
  Log(LOGLEVEL_DEBUG, " - maxOnDiskSegments:      %d", m_maxOnDiskSegments);
  Log(LOGLEVEL_DEBUG, " - currentDemuxTimeIndex:  %d", m_currentDemuxTimeIndex);
  Log(LOGLEVEL_DEBUG, " - minOnDiskSeekTimeIndex: %d", m_minOnDiskSeekTimeIndex);
```

Will also look to add a new function with e.g. `ADDON_LOG kodi::CurrentLogLevel()` to prevent not needed calls if not wanted level.

Reply:
It’s just a test PR for a user to debug something!